docs/database/database-metrics.md

```diff
@@ -440,6 +445,313 @@ This metric is [recommended][MetricRecommended].
 | [`db.client.connection.pool.name`](/docs/attributes-registry/db.md) | string | The name of the connection pool; unique within the instrumented application. In case the connection pool implementation doesn't provide a name, instrumentation SHOULD use a combination of parameters that would make the name unique, for example, combining attributes `server.address`, `server.port`, and `db.namespace`, formatted as `server.address:server.port/db.namespace`. Instrumentations that generate connection pool name following different patterns SHOULD document it. | `myDataSource` | `Required` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
 
 
+<!-- markdownlint-restore -->
+<!-- prettier-ignore-end -->
+<!-- END AUTOGENERATED TEXT -->
+<!-- endsemconv -->
+
+## Azure Cosmos DB
+
+The following metric instruments describe Azure Cosmos DB client behaviour.
+
+### Metric: `db.client.operation.duration`
+
+It captures the total time taken by an Azure Cosmos DB operation. This metric follows the common [db.client.operation.duration] (#metric-dbclientoperationduration)b definition.
+
+This metric SHOULD be specified with
+[`ExplicitBucketBoundaries`](https://github.com/open-telemetry/opentelemetry-specification/tree/v1.35.0/specification/metrics/api.md#instrument-advisory-parameters)
+of `[ 0.001, 0.005, 0.010,  0.050, 0.100, 0.200, 0.500, 1.000, 2.000, 5.000 ]`.
+
+Explaining bucket configuration:
+
+1. 0.001, 0.005, 0.010 seconds: : These boundaries provide high precision for high-performance workloads, especially when dealing with microservices or low-latency queries.
+2. 0.050, 0.100, 0.200 seconds: These intervals capture typical latencies between 50ms and 200ms, common in web applications, offering detailed tracking for normal operational performance.
+3. 0.500, 1.000 seconds: These boundaries identify operations that take longer, highlighting potential bottlenecks or slower requests that may need optimization.
+4. 2.000, 5.000 seconds: Although Azure Cosmos DB is optimized for sub-second latencies, these buckets are included to track rare, long-running outliers that might exceed 1 second.
+
+Refer `db.cosmosdb.operation.request_charge` metrics for dimensions.
+
+### Metric: `db.query.response.row_count`
+
+It captures the number of items returned by a query or feed operation in Azure Cosmos DB. It helps identify response sizes that may contribute to high latency, increased memory/CPU usage, or network call failures.
+
+This metric SHOULD be specified with
+[`ExplicitBucketBoundaries`](https://github.com/open-telemetry/opentelemetry-specification/tree/v1.35.0/specification/metrics/api.md#instrument-advisory-parameters)
+of `[10, 50, 100, 250, 500, 1000, 2000, 5000, 10000]`.
```

Comment:
any reason not to start from 1? I assume many queries always return 1 item.

Reply:
no its not true that many queries always return 1 item

Reply:
I believe it's part of DB query best practices to design database schemas and write queries to return one query, especially for simple things like `select from foo where id = <id>`.

I think it's important to know when it's the case and therefore important to include 1 as a lower boundary. And I don't see a  reason not to add it as a lower boundary.

Reply:
incuded 1

Reply:
Added 1 as new bucket.